qiskit_bot/config.py

```diff
@@ -62,21 +62,26 @@ def load_config(path):
 
 
 local_config_schema = vol.Schema({
-    'users': [{}],
-    'categories': vol.Optional({}),
+    vol.Optional('categories', default=default_changelog_categories): dict,
+    vol.Optional('notifications'): {vol.Extra: [str]},
+    vol.Optional('notification_prelude'): str,
+    vol.Optional('always_notify'): bool,
```

Comment:
This doesn't matter, just a question: are there any reasons to define `categories` with a default here, but leave the others default-less?

Reply:
If there is a sane implicit default value not really, just sloppy style on my part mostly. I think I might have had some trouble with voluptuous populating defaults at some point and just opted not to use it if I couldn't get it to work reliably (but that might have just been user error, I don't really remember)